servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -68,13 +68,9 @@ def update_job_labels(id, body, **kwargs):
 
     # Follow API spec
     result = response.json()
-    all_labels = get_job(id).labels
-    if not all_labels:
-        all_labels = {}
-
-    # Redundantly update all_labels with updated labels to provide consistency guarantees
-    all_labels.update(result.get('labels'))
-    return UpdateJobLabelsResponse(labels=all_labels)
+    if not result.get('labels'):
+        result.get('labels').update({})
```

Comment:
Is this still dependent on the cromwell update? If so you can probably leave this off for now.